dcos/cosmospackage.py

```diff
@@ -296,16 +299,34 @@ def remove_repo(self, name):
         response = self.cosmos_post("repository/delete", params=params)
         return response.json()
 
+    def experimental_package_add(self, dcos_package):
+        """
+        Adds a DC/OS package to DC/OS
+
+        :param dcos_package: path to the DC/OS package
+        :type dcos_package: str
+        :return: Response to the package add request
+        :rtype: Response
+        """
+        request = "add"
+        headers = self._request_preferences().get(request).pop(0)
+        headers['X-Dcos-Content-Length'] = str(_file_length(dcos_package))
```

Comment:
We don't need this anymore; it was a workaround to provide the content length for chunked requests.